app/ui/styles/components/menu-list.css

```diff
@@ -7,7 +7,7 @@
  */
 
 .ox-menu {
-  @apply max-h-[17.5rem] overflow-y-auto rounded border bg-raise border-secondary elevation-2;
+  @apply !max-h-[17.5rem] overflow-y-auto rounded border bg-raise border-secondary elevation-2;
```

Comment:
! just to make this stronger than some height the component is applying. But maybe there's a better way.